proxy.c

```diff
@@ -69,6 +72,7 @@ void forward_data_ext(int source_sock, int destination_sock, char *cmd);
 int parse_options(int argc, char *argv[]);
 
 int server_sock, client_sock, remote_sock, remote_port = 0;
+int active_connections = 0;
```

Comment:
You are absolutely right, you see, I was expecting to put the number of active connections, however, with all the forking this was hard and I reverted to how many connections were processed and didn't rename the vars, good catch :)